wagtail/wagtaildocs/templates/wagtaildocs/documents/confirm_delete.html

```diff
@@ -1,11 +1,19 @@
 {% extends "wagtailadmin/base.html" %}
-{% load i18n %}
+{% load wagtailadmin_tags i18n %}
 {% block titletag %}{% blocktrans with title=document.title %}Delete {{ title }}{% endblocktrans %}{% endblock %}
 {% block content %}
     {% trans "Delete document" as del_str %}
     {% include "wagtailadmin/shared/header.html" with title=del_str subtitle=document.title icon="doc-full-inverse" %}
 
     <div class="nice-padding">
+
+        {% usage_count_enabled as uc_enabled %}
```

Comment:
what's the purpose of this assignment?

Reply:
I don't know why, but just using `usage_count_enabled` doesn't work. This was also the case in the original code.

Reply:
As discussed with @m1kola: `{% usage_count_enabled as uc_enabled %}` assigns the result of the `usage_count_enabled` *tag* into the `uc_enabled` *value*, and this is a normal way to do that.

Reply:
Understood, thanks.